platform.md

```diff
@@ -872,6 +872,9 @@ To achieve reproducibility the lifecycle SHOULD set the following to a constant,
 - file modification times in generated layers
 - image creation time
 
+The platform MAY set `SOURCE_DATE_EPOCH`  in the lifecycle execution environment, where the value of `SOURCE_DATE_EPOCH` MUST be a [UNIX timestamp](https://reproducible-builds.org/specs/source-date-epoch/).
+If `SOURCE_DATE_EPOCH` is set, the lifecycle SHOULD set the image creation time to its value.
```

Comment:
I think setting just the image creation time is 👍 

The primary use case I've seen asked from time to time is mostly around the management of the produced images locally or in a registry. The image creation time should be enough to allow for automation around cleanup and auditing of produced images. 

I don't know that I've seen anyone reach out about the file mod times, but I would not be opposed to it either if someone felt strongly that we should honor `SOURCE_DATE_EPOCH` for the files inside the container as well.

Reply:
As the person who made the original Slack request, I can back up @jabrown85 here on the use case. 

This request is purely about how container registries don't really let us separate metadata like timestamp from the artifact itself which makes them a huge pain to manage. The concrete example being that I can tell Gitlab to keep the last 50 images we made of our main branch, but if they all have the same timestamp, then it removes all but 50 random images, often including the one we have currently deployed in prod! So all tag management basically becomes manual or scripted in a janky way like "check the last 50 tags in the git history for that branch and then remove anything that's not that."

I'd rather have a faster build time than not, since it the file mod time doesn't affect the end pain point.